cms/tests/test_apphooks.py

```diff
@@ -32,7 +32,7 @@
 APP_MODULE = "cms.test_utils.project.sampleapp.cms_apps"
 
 
-class ApphooksTestCase(CMSTestCase):
+class ApphooksTestCase(ClearURLs, CMSTestCase):
```

Comment:
I saw your post on the mailing list, but I think we need to get to the bottom of the URL issues. Many test cases already have `clear_url_caches()` calls in `setUp()` and `tearDown()` so I can't see why adding it in `setUpClass()` is needed. At least some comments explaining why so much URL cache clearing is needed would be quite helpful.